hooks/nuke_batch_render_movie.py

```diff
@@ -58,7 +59,7 @@ def __create_output_node(path, codec_settings, logger=None):
     return node
 
 
-def render_movie_in_nuke(path, output_path,
+def render_movie_in_nuke(path, output_path, extra_write_node_mapping,
                          width, height,
```

Comment:
Add the new parameter to the docstring. Also, it should really be added to the end.

Reply:
It just felt relevant to the path related parameters, should I move it to the end?